e2e/container/run_test.go

```diff
@@ -59,3 +59,12 @@ func createRemoteImage(t *testing.T) string {
 	icmd.RunCommand("docker", "rmi", image).Assert(t, icmd.Success)
 	return image
 }
+
+func TestRunWithCgroupNamespace(t *testing.T) {
+	environment.SkipIfDaemonNotLinux(t)
+	environment.SkipIfCgroupNamespacesNotSupported(t)
+
+	result := icmd.RunCommand("docker", "run", "--cgroupns=private", "--rm", fixtures.AlpineImage,
+		"sh", "-c", "[[ $(cat /proc/1/cgroup | grep memory | cut -d: -f 3) == '/' ]]")
```

Comment:
Perhaps rewrite this to not depend on bash? Also, looks like a single grep is sufficient, e.g.

`grep -q ':memory:/$' /proc/1/cgroup`

Reply:
@rgulewich ^^

Reply:
Updated - PTAL.

Reply:
@kolyshkin ^^